### DISCUSSION TITLE
Sound card not working error

### DISCUSSION THREAD

**Discussion by ajaz2009** [jamulussoftware/jamulus#2861] [Technical Support & Help]
I get the following error when trying to connect to the server I created. Anyone knows why?

![WhatsApp Image 2022-09-17 at 1 43 45 PM](https://user-images.githubusercontent.com/113794916/190869841-bdb7cd2d-863d-416d-9e95-49355d88a927.jpeg)

**Comment by jujudusud** [comment]
So you are on windows 10?
What is your selected sound card?
Is an ASIO driver installed or not?
